certchain/certmanager/augmentor.go

```diff
@@ -106,9 +106,24 @@ func (a *Augmentor) Start() error {
 	if err != nil {
 		return err
 	}
-	orNext, err := a.maintainOCSPResp()
-	if err != nil {
-		return err
+
+	var orNext futureevent.Event
+	retryCount := 0
+	const maxRetries = 3
```

Comment:
Maybe `maxRetryCount` (for consistency with `retryCount`). Alternative is to rename `retryCount` to `retries`, but I think `retryCount` is more descriptive.

Reply:
Done.